### PR TITLE
authentication note and edits

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1264,7 +1268,7 @@ In addition to time synchronization, Cassandra requires OS level configuration s
 
 ## Connecting Clients
 
-The only supported client for the DSOC Cassandra Service is the Datastax Java CQL Driver. Note that this means that Thrift RPC-based clients are not supported for use with this service and any legacy applications that use this communication mechanism are run at the user's risk.
+The only supported client for the DS/OS Cassandra Service is the Datastax Java CQL Driver. Note that this means that Thrift RPC-based clients are not supported for use with this service and any legacy applications that use this communication mechanism are run at the user's risk.
```
Comment (verma7) [inline README.md]: DC/OS?

File: README.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ cqlsh> SELECT * FROM demo.map;
 
 ## Install and Customize
 
+# About installing Cassandra on Enterprise DC/OS
+  
+ In Enterprise DC/OS `strict` [security mode](/1.8/administration/installing/custom/configuration-parameters/#security), Cassandra requires a service account. In `permissive`, a service account is optional. Only someone with `superuser` permission can create the service account. Refer to [Provisioning Cassandra](https://docs.mesosphere.com/1.8/administration/id-and-access-mgt/service-auth/cass-auth/#give-perms) for instructions.
```
Comment (mrbrowning) [inline README.md]: Missing the `https://docs.mesosphere.com` in this URL.

File: README.md [hunk 3]
```diff
@@ -303,11 +307,11 @@ $ curl -X POST -H "Authorization: token=$AUTH_TOKEN" http://<dcos_url>/service/c
 
 **If you are upgrading to or beyond 1.0.13-X.Y.Z of DC/OS Apache Cassandra from an older version (pre `1.0.13-X.Y.Z`), here is the upgrade path:**
 
-1. Perform [Backup Operation](https://github.com/mesosphere/dcos-cassandra-service#backup) on your currently running Cassandra Service. Please make a note of the backup name and backup location.
-2. [Install a new Cassandra Service](https://github.com/mesosphere/dcos-cassandra-service#multiple-cassandra-cluster-installation) instance.
-3. Perform [Restore operation](https://github.com/mesosphere/dcos-cassandra-service#restore) on the new cluster created in Step #2
-4. Once the restore operation is finished, check if the data is restored correctly.
-5. [Uninstall](https://github.com/mesosphere/dcos-cassandra-service#uninstall) old cluster.
+1. Perform [Backup Operation](https://github.com/mesosphere/dcos-cassandra-service#backup) on your currently running Cassandra Service. Note the backup name and backup location.
+1. [Install a new Cassandra Service](https://github.com/mesosphere/dcos-cassandra-service#multiple-cassandra-cluster-installation) instance.
```
Comment (mrbrowning) [inline README.md]: List numbers are all "1" here.

**Review by mrbrowning — COMMENTED** [review]
Looks good, just some minor fixes.

File: README.md (hunk 1 above)
Comment (sascala) [inline README.md]: :+1:

File: README.md (hunk 2 above)
Comment (sascala) [inline README.md]: 👍

**Pull request merged**